rosbag2_cpp/CMakeLists.txt

```
@@ -139,7 +139,9 @@ if(BUILD_TESTING)
     SHARED
     test/rosbag2_cpp/serializer_test_plugin.cpp
     test/rosbag2_cpp/converter_test_plugin.cpp)
-  ament_target_dependencies(converter_test_plugins rosbag2_storage)
+  target_link_libraries(converter_test_plugins
+    rosbag2_storage::rosbag2_storage
+  )
   target_link_libraries(converter_test_plugins ${PROJECT_NAME})
```

These two `target_link_libraries` could be combined together.
```suggestion
  target_link_libraries(converter_test_plugins
    ${PROJECT_NAME}
    rosbag2_storage::rosbag2_storage
  )
```